ecl/hqlcpp/hqlhtcpp.cpp

```diff
@@ -12975,8 +12982,16 @@ void HqlCppTranslator::doBuildAggregateProcessTransform(BuildCtx & ctx, BoundRow
             }
             break;
         }
+
+        if (isAggregateOp && isDynamicOffset)
```

Comment:
I don't mind if any work is done before an error condition is detected.
These other codes are checked in the case statements because the value being checked is different.  It could be refactored, but either is fine.